baip-0007.md

```diff
@@ -0,0 +1,58 @@
+2)The reward needs to be collected actively. If the reward is not collected within the time limit, this part of the reward will not be accumulated.The collection method can be to transfer any small number of bitasset to the specified account.
+3)If there are too few bitassets participating in the reward plan, set the upper limit of daily yield, such as 5 ‰.
+4)The more available bitassets you hold, the more rewards you get. Calculated as a minimum of 7 days. For example, if the number of bitCNY held within 7 days is 2000 3000 3000 1000 5000 5000 2000, then 1000 is the effective number. Note that the available bitassets do not include bitassets such as participating in Pending purchase\ compulsory liquidation and transfer.
+5)There is a reward for recommending new registered account to participate in lock in the reward plan. If the number of locked in bitassets in the account reaches a certain amount (e.g. 100000 bitcny, other bitassets are calculated based on the exchange rate year-on-year), it can be applied as a promotion account. If the new account recommended for registration by the account participates in the locked in of smart assets, the account can obtain corresponding rewards.
+
+### Regular destruction
+
+The dividend shall be destroyed at the same time as it is distributed in each reward cycle.
+Destruction per week = coefficient Z * (reward per week)
+
+### Parameter adjustment description
+
+For all parameters, the modification cycle is 2 weeks. Each modification can be adjusted for a function or a single parameter.
+Modification conditions (one of two options):
+
+1.Adopted by more than half of the members of the Council
+2.Send a simple baip, describe the parameters you want to modify, according to the passing standard of baip.
+
+# Continuous operation
+
+This proposal does not conflict with baip6(MM competition).
+      If baip6 passes, it will be carried out at the same time. After the consumption of the Council's stock funds, 30% of the Council's monthly income is used for destruction, 30% is used for warehouse locking and reward distribution, 30% is used for mining with hanging orders, and 10% is used for Council reservation. The parameters are adjusted in the same proportion.
+      If baip6 fails, the proposal will be implemented separately. After the consumption of the Council's stock funds, 30% of the Council's monthly income is used for destruction, 60% is used for lock in rewards, and 10% is used for the Council's reservation. The parameters are adjusted in the same proportion.
+
+# Organizer
+
+The organizer is appointed by the Council. In the early stage, a certain amount of funds will be spent by the Council account for program development, and 1% of the daily reward will be used as the organizer's salary after operation.
```

Comment:
What are the organizer tasks after starting operations? Why it requires 1% salary? I understand that initial development of script takes time, but continuous payments seems excessive. Think of fixed "operation fee".

Reply:
Every day, the organizer shall count, calculate and check the participation funds of the account, and issue rewards. According to the current price, 1% per day is about 30 yuan, a very low cost.

To emphasize, neither I nor CN vote will be the organizer, nor will we sign up for the organizer's campaign. The organizer will be determined by the Committee.

Reply:
Ok, add this info into BAIP.

Reply:
Who is the Council？the committee？
Why the development fund will come from the committee account not reserve pool？
How did the committee appoint the organizer？hwo to chose？
I think we have a very clear guideline on worker proposal pay，fee→committee account→reserve pool→worker proposal pay，if we didn't have this clear guideline，everyone can steal the communiy money easily, this is why we need a limit pay from reserve pool everyday，this is why we need refund400k.